src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -491,4 +498,26 @@ public static String replaceTokens(Map<String, ?> tokens, String value) {
 
         return buf.toString();
     }
+
+    public static void setupIDEResourceCopy(@Nonnull final Project project) {
+        if (project.getPlugins().hasPlugin(IdeaPlugin.class)) {
+            final IdeaPlugin idea = project.getPlugins().getPlugin(IdeaPlugin.class);
+            project.getTasks().create(CopyIDEAResources.NAME, CopyIDEAResources.class, task -> task.configure(idea.getModel()));
```

Comment:
Is it deliberate that this and the eclipse task eagerly creates\ the task rather than lazily registering them? If yes, add a comment why.